kin-sdk/kin-base/src/main/java/kin/base/Transaction.java

```diff
@@ -228,7 +228,14 @@ public static Transaction fromEnvelopeXdr(String envelope) throws IOException {
    */
   public static Transaction fromEnvelopeXdr(TransactionEnvelope envelope) {
     kin.base.xdr.Transaction tx = envelope.getTx();
+    // Although currently there couldn't be a transaction with no operations we still check it.
     int mFee = tx.getFee().getUint32();
```

Comment:
as a general rule, do your best to avoid using **m** prefix for local variables - it's confusing !

Reply:
Most of it is a stellar code whom we did not wrote.
Personally I am not using this prefix